deploy/sre-prometheus/100-curator-jobs.PrometheusRule.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-curator-jobs
+    role: alert-rules
+  name: sre-curator-jobs
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-curator-jobs
+    rules:
+    # https://issues.redhat.com/browse/OSD-6348
+    - alert: CuratorJobFailedSRE
+      expr: kube_job_failed{job="kube-state-metrics",namespace="openshift-logging",job_name=~"^curator-.*"} > 1
+      # The cronjob runs this once an hour; and (anecdotally) the job
+      # takes close to an hour to run.
+      for: 120m
```

Comment:
Should we bump this to 125m to allow the next job to run? When successful its duration is short. 

```oc -n openshift-logging get jobs                                                                       ✔    default/mta-dev-test/dofinn.openshift ⎈  !2/24  
NAME                                        COMPLETIONS   DURATION   AGE
curator-1612822500                          1/1           6s         46m
```

Reply:
Whoah, you ran one in 6s? The ones I've seen take ~50m.

The 2h takes into account that the cronjob runs them every 1h -- i.e. I'm giving the job a full hour to complete as well.